Dockerfile

```diff
@@ -2,22 +2,27 @@
 
 ARG FLAVOR=${TARGETARCH}
 
-ARG ROCMVERSION=6.1.2
+ARG ROCMVERSION=6.3.3
+ARG GOVERSION=1.23.4
+ARG CUDA12VERSION=12.8
+ARG CUDA11VERSION=11.3
```

Comment:
I kept these with the build stage that used them to remove duplication and simplify configuration